fpna/DataModel/DataStructure.ipynb

```diff
@@ -192,7 +256,7 @@
         {
             "cell_type": "code",
             "source": [
-                "public record ReferencedCashflow : SimpleCashflow",
+                "public record ReferencedCashflow : Cashflow, IReferenced",
```

Comment:
ReferencedCashflow should also inherit from IBased.
@sbatka should it also inherit from IBounded?

Reply:
yes agreed